lib/resty/core/regex.lua

```diff
@@ -126,6 +126,8 @@ ffi.cdef[[
                                              unsigned char *dst);
 
     uint32_t ngx_http_lua_ffi_max_regex_cache_size(void);
+
+    int ngx_http_lua_set_jit_stack_size(int size);
```

Comment:
This function name does not follow our public C API naming convention. We need the `ngx_http_lua_ffi_` prefix.